docs/user-guide/deferred-download.rst

```diff
@@ -193,6 +273,24 @@ following is a basic configuration with inline documentation::
 
 For more information about a configuration option, please consult the Squid documentation.
 
+If you choose to use Varnish instead of Squid, there are two configuration files to look at.
+The first is ``/etc/varnish/default.vcl``, where you will need to change the port Varnish
+proxies to::
+
+  # Default backend definition. Set this to point to the Pulp streamer, which is port 8751
+  # by default.
+  backend default {
+      .host = "127.0.0.1";
+      .port = "8751";
+  }
+
+The second is ``/etc/varnish/varnish.params`` where, among other things, the cache size
+and location is set.
```

Comment:
From my cursory test yesterday (I didn't tcpdump and inspect the request as it bounced from proxy to proxy), it does the right thing and pre-fetches the file. This was with Varnish 4.1.0.

One of Nectar's flaws is that it doesn't let us do Range requests, but if we fixed that (or stopped using Nectar), we wouldn't need that setting.